api/v1/storagecluster_types.go

```diff
@@ -48,8 +48,11 @@ type StorageClusterSpec struct {
 	StorageDeviceSets  []StorageDeviceSet                     `json:"storageDeviceSets,omitempty"`
 	MonPVCTemplate     *corev1.PersistentVolumeClaim          `json:"monPVCTemplate,omitempty"`
 	MonDataDirHostPath string                                 `json:"monDataDirHostPath,omitempty"`
-	MultiCloudGateway  *MultiCloudGatewaySpec                 `json:"multiCloudGateway,omitempty"`
-	NFS                *NFSSpec                               `json:"nfs,omitempty"`
+	// MgrActivePassive can be set as true to deploy 2 ceph manager pods, one active and one standby
+	// Ceph will promote the standby mgr when the active mgr goes down due to any reason
+	MgrActivePassive  bool                   `json:"mgrActivePassive,omitempty"`
```

Comment:
I think we should rename it to the `Enable ActivePassiveMgr`

Reply:
Since in the future there could also be other mgr settings, another idea is to define a struct for MgrSpec, and put this setting there. For example, the setting would then be: `mgr.enableActivePassive`

Reply:
+1

Reply:
done